docs/README.md

```diff
@@ -1,22 +1,21 @@
 # Data Quality Committee Rules
 
-The rules linked below are either available for public review or have been approved by the DQC.
+The rules linked below are have been approved.
 
-| Number | Short name | Status | Rule version |
+| Number | Short name | Status | Rule version (see [Guidance](../README.md)) |
 | ----- | ----- | ----- | ----- |
-| [DQC_US_0001](DQC_US_0001/DQC_0001.md) | Axis with inappropriate members | Approved | 2.0 |
-| [DQC_US_0004](DQC_US_0004/DQC_0004.md) | Element values are equal | Approved | 1.0 |
-| [DQC_US_0005](DQC_US_0005/DQC_0005.md) | Context dates after period end date | Approved | 1.1.0 |
-| [DQC_US_0006](DQC_US_0006/DQC_0006.md) | DEI and block tag date contexts | Approved | 1.0 |
-| [DQC_US_0009](DQC_US_0009/DQC_0009.md) | Element A must be less than or equal to element B | Approved | 1.0 |
-| [DQC_US_0013](DQC_US_0013/DQC_0013.md) | Negative values with dependence | Approved | 2.0 |
-| [DQC_US_0014](DQC_US_0014/DQC_0014.md) | Negative values with no dimensions | Approved | 2.0 |
-| [DQC_US_0015](DQC_US_0015/DQC_0015.md) | Negative values | Approved | 2.0 |
-| [DQC_US_0015_V2](DQC_US_0015/DQC_0015_V3.md) | Negative Values - Additions to Existing Rule DQC_0015 | Public Exposure Draft | 3.0.0 |
```

Comment:
@allenbross-wf - the row for DQC_US_0015_V3 Additions should be revised and still appear on the /docs/readme.md - below is the revision:
| [DQC_US_0015_V3](DQC_US_0015/DQC_0015_V3.md) | Negative Values - Additions to Existing Rule DQC_0015 | Public Exposure Draft | 3.0.0 |